components/lcd/esp_lcd_ssd1681/include/esp_lcd_panel_ssd1681.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include <stdbool.h>
+#include "esp_lcd_panel_vendor.h"
+#include "esp_lcd_panel_interface.h"
+#include "esp_lcd_panel_io.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+* @brief Prototype of ssd1681 driver event callback
+*
+* @param[in] handle    esp_lcd_panel_handle_t esp_lcd driver handle
+* @param[in] edata     reserved
+* @param[in] user_data User registered context, registered in `epaper_panel_register_event_callbacks()`
+*
+* @return Whether a high priority task is woken up by this function
+*/
+typedef bool (*esp_lcd_epaper_panel_cb_t)(const esp_lcd_panel_handle_t handle, const void *edata, void *user_data);
```

Comment:
Can we make the `const void *edata` a pointer to struct? How is the user supposed to know which event ocurred?

Reply:
This is a convention of callback functions in IDF. I think we had better stay consistent with other callback functions.